libpod/container_exec.go

```diff
@@ -1114,7 +1114,7 @@ func writeExecExitCode(c *Container, sessionID string, exitCode int) error {
 		return fmt.Errorf("syncing container %s state to remove exec session %s: %w", c.ID(), sessionID, err)
 	}
 
-	return justWriteExecExitCode(c, sessionID, exitCode)
+	return justWriteExecExitCode(c, sessionID, exitCode, false)
```

Comment:
these must be set true, no? Still PR still faces Doesn't this PR still introduce https://github.com/containers/podman/issues/20188 so this must be fixed before the merge to not cause regressions.

Reply:
passed true value as discussed.